src/libm-tester/CMakeLists.txt

```diff
@@ -37,9 +37,14 @@ macro(test_extension SIMD)
     add_dependencies(${TARGET_IUT${SIMD}} ${TARGET_HEADERS})
     add_dependencies(${TARGET_IUT${SIMD}} ${TARGET_LIBSLEEF})
 
-    add_test(NAME ${TARGET_IUT${SIMD}}
-      COMMAND ${TARGET_TESTER} ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${TARGET_IUT${SIMD}})
-    
+    if (SDE_COMMAND)
+      add_test(NAME ${TARGET_IUT${SIMD}}
+	COMMAND ${TARGET_TESTER} "--sde" ${SDE_COMMAND} ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${TARGET_IUT${SIMD}})
+    else()
```

Comment:
Please use the form:

```
if (SDE_COMMAND)
// set SDE_RUN to be "--sde" ${SDE_COMMAND}
else()
// set SDE_RUN to be "" (empty)
endif()
add_test(NAME ${TARGET_IUT${SIMD}}
COMMAND ${TARGET_TESTER} ${SDE_RUN} ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${TARGET_IUT${SIMD}})
```

You could move the if statement at global configuration when detecting the SDE_COMMAND.